_internals/software-architecture.md

```diff
@@ -5,10 +5,10 @@ nav_order: 1
 use_mermaid: true
 ---
 
-Babelfish implements Microsoft SQL Server behavior on top of PostgreSQL. To achieve this goal, a
-couple of areas had to be modified inside the PostgreSQL core. PostgreSQL does
-not offer this capability out-of-the-box, therefore add-ons had to be provided
-to make this happen.
+Babelfish implements Microsoft SQL Server behavior on top of PostgreSQL.
+To achieve this goal, the server had to be modified.  This was mostly done
```

Comment:
Entire paragraph can simply be:

This is implemented as a set of [extensions](https://www.postgresql.org/docs/current/extend-extensions.html). The [Babelfish extensions](https://github.com/babelfish-for-postgresql/babelfish_extensions) do require a [slightly modified version of PostgreSQL](https://github.com/babelfish-for-postgresql/postgresql_modified_for_babelfish).

Reply:
Done. I couldn't get myself to include "slightly"; that would be more than a white lie.